datasets/ler/ler.py

```diff
@@ -0,0 +1,166 @@
+                  Graphs. Proceedings of the 15th International Conference
+                  (SEMANTiCS 2019)},
+  year = 2019,
+  editor = {Maribel Acosta and Philippe Cudré-Mauroux and Maria
+                  Maleshkova and Tassilo Pellegrini and Harald Sack and York
+                  Sure-Vetter},
+  keywords = {aip},
+  publisher = {Springer},
+  series = {Lecture Notes in Computer Science},
+  number = {11702},
+  address = {Karlsruhe, Germany},
+  month = 9,
+  note = {10/11 September 2019},
+  pages = {272--287},
+  pdf = {https://link.springer.com/content/pdf/10.1007%2F978-3-030-33220-4_20.pdf}
+}
+"""
+
+_DESCRIPTION = """\
+We describe a dataset developed for Named Entity Recognition in German federal court decisions. 
+It consists of approx. 67,000 sentences with over 2 million tokens. 
+The resource contains 54,000 manually annotated entities, mapped to 19 fine-grained semantic classes: 
+person, judge, lawyer, country, city, street, landscape, organization, company, institution, court, brand, law, 
+ordinance, European legal norm, regulation, contract, court decision, and legal literature. 
+The legal documents were, furthermore, automatically annotated with more than 35,000 TimeML-based time expressions. 
+The dataset, which is available under a CC-BY 4.0 license in the CoNNL-2002 format, 
+was developed for training an NER service for German legal documents in the EU project Lynx.
+"""
+
+_URL = "https://raw.githubusercontent.com/elenanereiss/Legal-Entity-Recognition/master/data/ler.conll"
+
+
+class Ler(datasets.GeneratorBasedBuilder):
+    """
+    We describe a dataset developed for Named Entity Recognition in German federal court decisions.
+    It consists of approx. 67,000 sentences with over 2 million tokens.
+    The resource contains 54,000 manually annotated entities, mapped to 19 fine-grained semantic classes:
+    person, judge, lawyer, country, city, street, landscape, organization, company, institution, court, brand, law,
+    ordinance, European legal norm, regulation, contract, court decision, and legal literature.
+    The legal documents were, furthermore, automatically annotated with more than 35,000 TimeML-based time expressions.
+    The dataset, which is available under a CC-BY 4.0 license in the CoNNL-2002 format,
+    was developed for training an NER service for German legal documents in the EU project Lynx.
+    """
+    VERSION = datasets.Version("1.0.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("int32"),
+                    "tokens": datasets.Sequence(datasets.Value("string")),
+                    "ner_tags": datasets.Sequence(
```

Comment:
Nice ! That's exactly the format we expect for NER :)

Reply:
great, thanks :)